lib/utils/cookie.js

```diff
@@ -33,19 +34,19 @@ cookie.prototype.get = function(str){
 };
 
 cookie.prototype.set = function(str, value){
```

Comment:
Re: double parsing when necessary, I tend to lean more toward minimizing
the upgrade impact and keeping the mess internal to the SDK, rather than in
users' code. It's not an ideal pattern in most cases, but in this case it's
a minimal change and might be better than introducing a workaround for
early adopters to keep access to their data.
On Thu, Mar 24, 2016 at 12:54 PM Eric Anderson <notifications@github.com>
wrote:

> In lib/utils/cookie.js
> <https://github.com/keen/keen-tracking.js/pull/41#discussion_r57377698>:
>
> > @@ -33,19 +34,19 @@ cookie.prototype.get = function(str){
> >  };
> >
> >  cookie.prototype.set = function(str, value){
>
> Great idea!!!
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/keen/keen-tracking.js/pull/41/files/b3687d22dce22e3eecda8cac4096159aab9e626b#r57377698>
>